packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -673,7 +676,9 @@ export const activate = async (extensionContext: vscode.ExtensionContext) => {
       TelemetryService,
       WorkspaceContext,
       CommandEventDispatcher
-    }
+    },
+    getCommandLog,
+    getLastCommandLogEntry
```

Comment:
These seems they they should be wrapped in a service vs just exported as standalone function to following the existing shared services pattern.  CommandService on like 679 instead?

Reply:
Good idea I'll give that a whorl.